test/util/all_close_f.cpp

```diff
@@ -361,7 +364,7 @@ bool test::all_close_f(const vector<double>& a, const vector<double>& b, int tol
     }
     if (!rc)
     {
-        NGRAPH_INFO << "diff count: " << diff_count << " out of " << a.size();
+        ar_fail << "diff count: " << diff_count << " out of " << a.size() << "\n";
```

Comment:
Remove `if (!rc)` which is still using the old technique for limiting error reporting.